<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CONTROL</class>
 <widget class="QMainWindow" name="CONTROL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>210</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/background/image/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widgettime" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>271</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#widgettime{border-image: url(:/background/image/2.png);}</string>
    </property>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>231</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>8</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Dec</enum>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="datelcdNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255,0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>10</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="weathericonlabel">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>71</width>
      <height>54</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="weatherlabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>320</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="horizontalSpacing">
      <number>30</number>
     </property>
     <property name="verticalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="1">
      <widget class="QPushButton" name="photoButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/icon/image/iocn/f.png);
border:0px;
background-color: rgba(255, 255, 255,0.9);
border-radius:15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="calendarButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/icon/image/iocn/r.png);
border:0px;
background-color: rgba(255, 255, 255,0.9);
border-radius:15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="adminButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/icon/image/iocn/a.png);
border:0px;
background-color: rgba(255, 255, 255,0.9);
border-radius:15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="controlButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/icon/image/iocn/z.png);
border:0px;
background-color: rgba(255, 255, 255,0.9);
border-radius:15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="cameraButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/icon/image/iocn/x.png);
border:0px;
background-color: rgba(255, 255, 255,0.9);
border-radius:15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="musicButton">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/icon/image/iocn/y.png);
border:0px;
background-color: rgba(255, 255, 255,0.9);
border-radius:15px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="calendwidget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>451</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>401</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 11pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="gridVisible">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QCalendarWidget::SingleSelection</enum>
     </property>
     <property name="horizontalHeaderFormat">
      <enum>QCalendarWidget::ShortDayNames</enum>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icon/image/iocn/close.png);
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="citylineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>161</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px;
border-radius:10px;
background-color: rgb(238, 238, 236);</string>
     </property>
     <property name="placeholderText">
      <string>请输入要查询的城市</string>
     </property>
    </widget>
    <widget class="QPushButton" name="weatherButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/icon/image/iocn/sou.png);
border:none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
image: url(:/icon/image/iocn/quit.png);
border-radius:15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListView" name="hotlistView">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>51</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:30px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>29</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>微博热搜</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ailineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(114, 159, 207);
border-radius:15px;
border:2px;</string>
    </property>
    <property name="placeholderText">
     <string>小爱AI助手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="aipushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/icon/image/iocn/send.png);
background-color: rgb(255, 255, 255);
border-radius:15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ailabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>widgettime</zorder>
   <zorder>weathericonlabel</zorder>
   <zorder>weatherlabel</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButton</zorder>
   <zorder>hotlistView</zorder>
   <zorder>label_2</zorder>
   <zorder>ailineEdit</zorder>
   <zorder>aipushButton</zorder>
   <zorder>ailabel</zorder>
   <zorder>calendwidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
